modules/services_cloud_bench/main.tf

```diff
@@ -0,0 +1,90 @@
+resource "sysdig_secure_cloud_account" "cloud_account" {
+  account_id = var.account_id
+  cloud_provider = "aws"
+  role_enabled = "true"
+}
+
+resource "aws_iam_role" "cloudbench_role" {
+  name = "SysdigCloudBenchRole"
+  assume_role_policy = data.aws_iam_policy_document.trust_relationship.json
+  tags = var.tags
+}
+
+data "aws_iam_policy_document" "trust_relationship" {
+  statement {
+    effect = "Allow"
+    actions = ["sts:AssumeRole"]
+    principals {
+      type = "AWS"
+      identifiers = ["arn:aws:iam::059797578166:user/noah.kraemer"] // TODO fix this
```

Comment:
This needs to be updated once we provision an internal user that cloud-bench will use.

Reply:
Yep, that's a great idea! I think it makes sense to use a datasource from the `sysdig_secure_cloud_account` provider, since I don't think we want to allow anything except the few known sysdig accounts. And yes, these will likely be different for different SAAS regions but will be the same for all customers